cartridges/int_algolia/cartridge/configuration/productFilterConfig.example.js

```diff
@@ -0,0 +1,49 @@
+// Product Filter Configuration Example File
+//
+// This configuration allows you to control which products are indexed to Algolia.
+// By default, products must be online, searchable, and have at least one online category.
+//
+// To customize the filtering behavior:
+//   - Create a `productFilterConfig.js` file in this directory
+//   - Upload the cartridge
+//   - The filters will be applied during product indexing
+//
+// The `productFilterConfig.js` file should export an object with the following structure:
+
+const productFilterConfig = {
+    // Control whether to check if products are online
+    // Default: true (only index online products)
+    checkOnline: true,
```

Comment:
It's not clear what this boolean will do. Users will understand that it will "check the online status", but it doesn't indicate what happens next. I think we can make them more explicit.
What about `includeOfflineProducts`?
Same below: something like `includeNotSearchableProducts`, `includeProductsWithNoOnlineCategory` (those names are probably not perfect, but they are at least explicit, let me know if you have a better idea)